ui/package.json

```diff
@@ -6,7 +6,7 @@
   "repository": "https://github.com/Apicurio/apicurio-registry.git",
   "license": "Apache-2.0",
   "private": true,
-  "federatedModuleName": "apicurio_registry",
+  "federatedModuleName": "sr",
```

Comment:
This probably isn't right - instead we should use the `apicurio_registry` name in apps which consume the service registry...

Reply:
@pmuir Then we need to do changes in  `config`  for `appication-services-ui` and  `ui-shared`
Should I update `sr`=>`apicurio_registry`  in `appication-services-ui` and  `ui-shared`  ? Please confirm.

Reply:
Yes sounds good. You'll also need to use it in `FederatedModule` components.